### PR TITLE
Add code tabs for _overviews/scala3-book/types-union.md

### PR DESCRIPTION
**Pull request by jglodek**
Regarding https://github.com/scala/docs.scala-lang/issues/2481
made on Warsaw Scala Spree

image

### CODE REVIEW

File: _overviews/scala3-book/types-union.md [hunk 1]
```diff
@@ -23,59 +28,109 @@ def help(id: Username | Password) =
...
+
 ### Alternative to Union Types
 As shown, union types can be used to represent alternatives of several different types, without requiring those types to be part of a custom-crafted class hierarchy, or requiring explicit wrapping.
 
 #### Pre-planning the Class Hierarchy
 Other languages would require pre-planning of the class hierarchy, like the following example illustrates:
 
+{% tabs union-preplanning %}
+
+{% tab 'Scala 3 Only' %}
```
Comment (bishabosha) [inline _overviews/scala3-book/types-union.md:75]:
```suggestion
{% tab 'Scala 2 and 3' %}
```

File: _overviews/scala3-book/types-union.md [hunk 2]
```diff
@@ -23,59 +28,109 @@ def help(id: Username | Password) =
...
 
+{% endtab %}
+
+{% endtabs %}
+
 ### Alternative to Union Types
 As shown, union types can be used to represent alternatives of several different types, without requiring those types to be part of a custom-crafted class hierarchy, or requiring explicit wrapping.
 
 #### Pre-planning the Class Hierarchy
 Other languages would require pre-planning of the class hierarchy, like the following example illustrates:
```
Comment (bishabosha) [inline _overviews/scala3-book/types-union.md:71]:
```suggestion
In other languages, such as Scala 2, you would require pre-planning of the class hierarchy, like the following example illustrates:
```

**Review by bishabosha — CHANGES_REQUESTED** [review]
Thanks for contributing!, just a minor point about "pre planning the class hierarchy" is meant to be a comparison to other languages, so scala 2 can be considered here

**Comment by julienrf** [comment]
Hey @jglodek, thank you for opening this PR.

I went ahead and migrated the file types-union.md in #2768, so I am closing this PR.

**Pull request closed without merging**